CHANGELOG.md

```diff
@@ -2,8 +2,10 @@
 
 ------ FOLLOW THESE WHILE ADDING AN ENTRY ------
 
-** Add BREAKING keyword in bold for changes which could potentially break the component, eg: **BREAKING**
-** Enclose a prop name in double backticks, eg: `isLoading`
+** Add BREAKING keyword in bold for changes which could potentially break the component, eg: **BREAKING**.
+** Represent a component name in italics, eg: _Modal_.
+** Enclose a prop name in double backticks, eg: `isLoading`.
```

Comment:
change the `isLoading` prop to some prop in neetoEditor. Do the same in PR template file as well.